market_server/src/cli.rs

```diff
@@ -0,0 +1,14 @@
+use clap::Parser;
+
+use crate::Port;
+
+#[derive(Debug, Parser)]
+#[command(author, version, about, long_about = None)]
+pub struct Cli {
+    #[arg(short, long, default_value = "8080")]
```

Comment:
Default port for the market was previously established to be: 50051
Should modify this for consistency